src/js/Dropdown/index.ts

```diff
@@ -2,7 +2,7 @@ import Control, { ControlProps } from '../Control';
 import Message from '../../constant/Message';
 import Item from './Item';
 import AbstractSingleSelection from '../utils/AbstractSingleSelection';
-import { mdiChevronDown } from '@mdi/js'
+import { mdilChevronDown } from '@mdi/light-js'
```

Comment:
Do you need to use @mdi/light-js?
If so, please register on SSR task this library and notify Hoa-san for security checking.
Thanks!